corehq/apps/export/models/new.py

```diff
@@ -1159,3 +1207,9 @@ def get_value(self, doc, base_path, transform_dates=False, row_index=None):
             [".".join([unicode(i) for i in row_index])]
             + (list(row_index) if len(row_index) > 1 else [])
         )
+
+
+# These must match the constants in corehq/apps/export/static/export/js/const.js
+MAIN_TABLE = []
```

Comment:
`CASE_HISTORY_TABLE` and `PARENT_CASE_TABLE` had to be moved so they could reference `PathNode` without circular imports, and I thought it made sense to move `MAIN_TABLE` along with them. But I don't feel strongly about it.